<launch>
    <rosparam file = "$(find olders_helper)/param/real_goals.yaml" />
    <node pkg="olders_helper" type="robot_action" name="robot_action_node" output="screen" />
    <node pkg="olders_helper" type="audio_server" name="audio_server" output="screen"/>
    <node pkg="olders_helper" type="action_client" name="action_client" output="screen"/>

</launch>